hudhook/src/hooks/dx9.rs

```diff
@@ -51,12 +51,19 @@ unsafe fn draw(this: &IDirect3DDevice9) {
                 imgui_wnd_proc as usize as i32,
             ));
 
-            Mutex::new(Box::new(ImguiRenderer {
+            // Create the imgui rendererer
+            let mut imgui_renderer = ImguiRenderer {
                 ctx: context,
                 renderer,
                 wnd_proc,
                 flags: ImguiRenderLoopFlags { focused: false },
-            }))
+            };
+
+            // Initialize window events on the imgui renderer
+            ImguiWindowsEventHandler::setup_io(&mut imgui_renderer);
```

Comment:
@FrankvdStam I haven't tested DX9 much; do you think this is appropriate here, in the same way as DX11/12? Looks good to me but not sure if you left it out on purpose.

Reply:
Awaiting reply here is probably the best yep

Reply:
Nice seeing an openGL target being added, nice work Jakobzs!
As for this line of code, I'm honestly not sure, I still didn't find the time to actually use the dx9 target in a real application. Did you actually try, and had problems, leading to you adding this line?

Reply:
Thank you :D

I have honestly not tested it on a DX9 target as I do not know one currently (all I know are DX11 and up). But from reasoning of that it was an issue in this hook (hence the added code), and also I based a lot of it on yours, then I assumed it might have been an issue there as well. It is a bit out of scope of this pull request tbh, but could be a nice fix if it is really needed.

Reply:
If these are window events like resizing imgui windows, then that is already working without this.  There's at least 1 bug that I know of, which is that the game will crash when changing the resolution. Not sure if this could be related.

Maybe its better to just leave this alone until one of us actually uses it in a dx9 game, thats' when we'll find all the bugs.

Reply:
Could probably be ideal yeah. I will just revert it.